randovania/games/samus_returns/json_data/Area 8.txt

```diff
@@ -979,7 +979,35 @@ Extra - asset_id: collision_camera_022
   * Layers: default
   * Extra - start_point_actor_name: ST_Queen_001_Checkpoint
   > Event - Queen Metroid
-      Trivial
+      All of the following:
+          Any of the following:
+              # Energy Requirements
+              Combat (Advanced) or Normal Damage ≥ 599
```

Comment:
no hypermode combat?

Reply:
I didn't add everything since the goal was to add some logic, and I didn't feel like testing every situation right now.

Reply:
fair nuff, they can be made more extensive later